docs/controls/access_control/ac-05.md

```diff
@@ -0,0 +1,53 @@
+  * Data Prepper *SuperUser* and *Staff* flags are both set to *False*.  Data Preppers do not have access to Django Admin, cannot provision new TDP accounts, and do not have access to the account management functionality.
+
+Developers of the TDP application are responsible for the software development and maintenance of the system.
+  * Developers are granted only enough permission in GitHub and Cloud.gov to support their duties.    
+  * All code and documentation committed to HHS has to be approved by the Product Manager and the HHS Tech Lead.
+
+The System Owner would have *Staff* and *SuperUser* access along with the OFA Admin role and has the ability to manage TDP accounts, including provisioning new user accounts on request. Other users with the role OFA Admin would not have *Staff* and *SuperUser* which separates the System Owner from OFA Admin.  
+
+
+**Part b.**  
+
+**Developers**  
+Developers are granted only enough permission in GitHub and Cloud.gov in order to support their duties.  All code and documentation committed to HHS has to be approved by the Product Manager and the HHS Tech Lead.  
+
+**Users (OFA Admin and Data Preppers)**
+The OFA Admin approves new users, updates profile information, deactivates, and reactivates users (Data Preppers (STTs) do not have access to this).  Additionally, OFA admins have the ability to upload data on behalf of Data Preppers and upload data files locally into the web application. 
+
+Users from the states, tribes, and territories (STT) who will be uploading data will have the role Data Preppers.  For the OFA MVP, STT will not have access to the TDP system, but OFA Admin will act as Data Prepper roles. Data Preppers  are able to upload new TANF reports, replace and resubmit TANF reports, and download their uploaded reports. 
+
+**Part c.**   
+When new users go to the TDP landing page, they must click on the "Sign in with Login.gov" button.  Once they click on that button, the user is redirected to the Login.gov website.  To gain access to the TDP system, users must first create an account at Login.gov.  By creating an account in Login.gov, the user's profile is created within TDP and they have access to the TDP system without user functionality.  After they are authenticated with Login.gov, the user is redirected to the TDP frontend landing page.  Once a user is logged into the TDP system, they can request user functionality by submitting their information through the request form.  The OFA Admin can view the request through the Django Admin and grants the user the appropriate permissions based on their job responsibilities.  
+
+***Access to the Django admin interface***  
```

Comment:
Line 44 through the end of the doc feel unnecessary since we've covered these details already in the previous sections. For conciseness, what do you think about removing `Access to the Django admin interface` and `Access to the TDP system`?

Reply:
I agree.  Removed both of these sections.